ci-operator/step-registry/ipi/deprovision/vsphere/diags/ipi-deprovision-vsphere-diags-commands.sh

```diff
@@ -16,6 +16,20 @@ source "${SHARED_DIR}/vsphere_context.sh"
 
 function collect_diagnostic_data {
   set +e
+  declare install_failed
+
+  # Check if install or bootstrap failed
+  if test -f "${SHARED_DIR}/install-status.txt"
+  then
+    EXIT_CODE=$(cat ${SHARED_DIR}/install-status.txt | awk '{print $1}')
```

Comment:
lol, why did I add this

Will fix after jobs run since so many started concurrently.